src/Moat/Class.hs

```diff
@@ -117,6 +118,9 @@ instance {-# OVERLAPPABLE #-} forall a. ToMoatType a => ToMoatType [a] where
 
 instance {-# OVERLAPPING #-} ToMoatType [Char] where toMoatType _ = Str
 
+instance forall a. ToMoatType a => ToMoatType (Set a) where
+  toMoatType _ = Array (toMoatType (Proxy @a))
```

Comment:
Maybe there's a more appropriate set type for ios and android that can be parsed from a json list? @kylebshr @tadfisher

Reply:
On Android it's just `Set<A>`, so the existing conversion works as is.